torchvision/prototype/transforms/_augment.py

```diff
@@ -86,7 +86,7 @@ def _get_params(self, sample: Any) -> Dict[str, Any]:
         return dict(i=i, j=j, h=h, w=w, v=v)
 
     def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
-        if isinstance(inpt, (features.Image, torch.Tensor)):
+        if is_simple_tensor(inpt) or isinstance(inpt, features.Image):
```

Comment:
FYI: Everywhere you currently do isinstance(..., Image) will need to change to support Videos. We don't have to act upon this now, but something to keep in mind.